scripts/pre_commit_linter.py

```diff
@@ -330,14 +331,19 @@ class FileCache(object):
 
     @classmethod
     def read(cls, filename, mode='r'):
+        """Returns the data read from the file."""
```

Comment:
Here and below, missing Args and Returns sections

Reply:
Done